Dockerfile.test

```diff
@@ -1,3 +1,4 @@
 FROM davidtnfsh/pycon_etl:cache
 ENV AIRFLOW_TEST_MODE=True
-USER root
\ No newline at end of file
+USER root
+COPY fixtures/airflow.test.cfg airflow.cfg
```

Comment:
Why do we put it in `fixtures`? Is it an airflow convention?

Reply:
no it's just my convention 😅 

any suggestion to put these kinds of testing config?

Reply:
I would simply put this file in the root directory. Just curious. Why use the word `fixture`?

Reply:
Sure, will put it in the root folder.
Just used to put every test-related file into fixtures folder, but okay with either